### ISSUE TITLE
[Bug]: After upgrade milvus version, the index created before does not exist and the search result is empty

### ISSUE THREAD

**Issue by ThreadDao** [milvus-io/milvus#18994]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: v2.1.1 -> master-20220902-4c1e7bc
- Deployment mode(standalone or cluster): cluster
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus==2.2.0.dev23
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

1. Deploy milvus cluster with image `v2.1.1`
2. Insert entities 20000, dim 128
3. Create index successfully `default_index_params = {"index_type": "IVF_SQ8", "metric_type": "L2", "params": {"nlist": 64}}`
4. Upgrade milvus image to `master-20220902-4c1e7bc`
- collection.has_index -> False
- collection.search -> []

```
<name>: upgrade_index
<partitions>: [{"name": "_default", "collection_name": "upgrade_index", "description": ""}]
<description>: 
<schema>: {
  auto_id: True
  description: 
  fields: [{
    name: int64
    description: 
    type: 5
    is_primary: True
    auto_id: True
......  (api_request.py:31)
[2022-09-02 07:16:44,637 - DEBUG - ci_test]: (api_request)  : [Collection.flush] args: [], kwargs: {'timeout': 60} (api_request.py:56)
[2022-09-02 07:16:44,638 - DEBUG - ci_test]: (api_response) : None  (api_request.py:31)
[2022-09-02 07:16:44,639 - DEBUG - ci_test]: 20000 (test_upgrade_argo.py:45)
[2022-09-02 07:16:44,639 - DEBUG - ci_test]: (api_request)  : [Collection.has_index] args: [], kwargs: {'index_name': '_default_idx'} (api_request.py:56)
[2022-09-02 07:16:44,640 - DEBUG - ci_test]: (api_response) : False  (api_request.py:31)
[2022-09-02 07:16:44,640 - DEBUG - ci_test]: (api_request)  : [Collection.load] args: [None, 1, 20], kwargs: {} (api_request.py:56)
[2022-09-02 07:16:45,175 - DEBUG - ci_test]: (api_response) : None  (api_request.py:31)
[2022-09-02 07:16:45,175 - DEBUG - ci_test]: (api_request)  : [Collection.search] args: [[[0.14701379211600396, 0.3989620084562091, 0.7996638759190872, 0.941484128417578, 0.37531906769952394, 0.5676843622375805, 0.9428731805286313, 0.327868333994245, 0.5852583438823732, 0.5797026516490473, 0.5002599775366443, 0.4387579831604075, 0.6210883997743637, 0.8178320782755679, 0.720826750365, 0......, kwargs: {} (api_request.py:56)
[2022-09-02 07:16:45,401 - DEBUG - ci_test]: (api_response) : ['[]', '[]']  (api_request.py:31)
FAILED
```
client:
- cluster: `devops`
- namespace: `qa`
- pod: `cluster-upgrade-index-before-fwtjt-674834778`

milvus:
- cluster: `devops`
- namespace: `chaos-testing`
- pod before upgrade
```
cluster-upgrade-milvus-datacoord-5879f54686-8nndx    1/1     Running   0          31m
cluster-upgrade-milvus-datanode-57c56fc45f-88shs     1/1     Running   0          31m
cluster-upgrade-milvus-indexcoord-f9b9d59b8-w7r2d    1/1     Running   0          31m
cluster-upgrade-milvus-indexnode-5494c5666f-ljx85    1/1     Running   0          31m
cluster-upgrade-milvus-proxy-6d4bc6cd5c-pwhkt        1/1     Running   0          31m
cluster-upgrade-milvus-querycoord-5dd89b9dbd-g4bp8   1/1     Running   0          31m
cluster-upgrade-milvus-querynode-74fc9dd8d9-w4846    1/1     Running   0          31m
cluster-upgrade-milvus-rootcoord-6c5894b6b-ft8zw     1/1     Running   0          31m
```
- pod after upgrade
```
pod/cluster-upgrade-milvus-datacoord-75b9c6fdf4-k2hsb condition met
pod/cluster-upgrade-milvus-datanode-77c9985dbc-27qs4 condition met
pod/cluster-upgrade-milvus-indexcoord-74b5849c6-hjxr6 condition met
pod/cluster-upgrade-milvus-indexnode-7c68b6fc49-m2vlp condition met
pod/cluster-upgrade-milvus-proxy-7cf88978f-n5znq condition met
pod/cluster-upgrade-milvus-querycoord-67cdc9998d-f6tl8 condition met
pod/cluster-upgrade-milvus-querynode-596b9b96fc-mkkvm condition met
pod/cluster-upgrade-milvus-rootcoord-cfc86bd78-9m7x9 condition met
```

### Expected Behavior

1. Index exist
2. Search return yopk result

**Comment by longjiquan** [comment]
Hello, @ThreadDao @zhuwenxing , I think this issue was already fixed by #19709 . Please help to recheck this, thx.
/assign @ThreadDao @zhuwenxing

**Comment by ThreadDao** [comment]
Fixed
milvus image: master-20221102-51acba7d
meta-migration image: 20221101-f2f10cb11
